ee/packages/amplication-git-pull-service/.env

```diff
@@ -19,11 +19,11 @@ KAFKA_CLIENT_ID=repository-pull
 KAFKA_REPOSITORY_PUSH_QUEUE="git.external.push.event.0"
 
 # Git
-STORAGE_PATH=[path-to-local-folder]
+STORAGE_PATH=.temp
```

Comment:
based on @arielweinberger suggestion - it should be .amplication_temp 
please see this comment on the original issue 
https://github.com/amplication/amplication/issues/2969#issuecomment-1166203987